contents/docs/new-to-posthog/revenue.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "See the money: Capturing revenue with PostHog"
+---
+
+import { AnalyticsPlannerTip } from 'components/Docs/AnalyticsPlannerTip'
+import { Admonition } from 'components/Docs/Admonition'
+
+<AnalyticsPlannerTip />
+
+While the analytics tasks so far have been full of subtlety, revenue is easy.
+
+It's binary.
+
+Either someone has given us money or they have not.
+
+There are two options for tracking revenue in PostHog.
+
+### Option 1 – Manual revenue capture
+
+Gathering data about revenue can be straightforward: you know exactly when the event happens, and hopefully you can back out from that event to imagine the precursor events you should also measure.
+
+Start with the place in the product where money changes hands. Put an event there. In addition, capture metadata using event properties: how much money the customer paid and what they paid for. 
+
+If you're selling subscriptions, record the subscription timeframe. If you're selling products, include the products they purchased. For example:
+
+```
+subscription_purchased  
+months: 12  
+price: 19.99
+```
+
+Try to keep property values strictly numeric – `months: 12` instead of `term: 12_months` – to preserve flexibility in building reports later on.
+
+Even if we don't have a lot of variation in what we're selling at first, record the details now so that later expansions, experiments or sales compare easily with historical data.
```

Comment:
```suggestion
Even if you don't have a lot of variation in what you're selling at first, record the details now so that later expansions, experiments, or sales compare easily with historical data.
```